node/mo/utils/db.ts

```diff
@@ -77,10 +66,24 @@ export async function openLinksCollection(db: Db): Promise<Collection<LinkDocume
  * @param fn The handler that works with the links data
  * @returns A promise that returns the result of the handler
  */
-export async function withLinksCollection<T>(fn: (links: Collection<LinkDocument>) => T): Promise<Awaited<T>> {
+export async function withLinksCollection<T>(fn: (links: Collection<MoLink>) => T): Promise<Awaited<T>> {
     return await withClient(async (client) => {
         const db = await openDefaultDB(client);
         const coll = await openLinksCollection(db);
         return await fn(coll);
     });
 }
+
+/**
+ * Convert an AsyncIterable into an array of objects
+ * @param cur An asynchronous iterable object
+ * @returns An array of all iterated elements
+ * @note Beware arbitrary large iterables!
+ */
+export async function getAll<T>(cur: AsyncIterable<T>): Promise<T[]> {
```

Comment:
Can we use toArray() instead https://www.mongodb.com/docs/drivers/node/current/fundamentals/crud/read-operations/cursor/#return-an-array-of-all-documents

Reply:
Oh, that's easier.